packages/apps/human-app/server/src/modules/oracle-discovery/oracle-discovery.controller.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  Body,
+  Controller,
+  Get,
+  UsePipes,
+  ValidationPipe,
+} from '@nestjs/common';
+import { Mapper } from '@automapper/core';
+import { InjectMapper } from '@automapper/nestjs';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+import { OracleDiscoveryService } from './oracle-discovery.serivce';
+import {
+  OracleDiscoveryCommand,
+  OracleDiscoveryResponse,
+  OracleDiscoveryDto,
+} from './interface/oracle-discovery.interface';
+
+@Controller()
+export class OracleDiscoveryController {
+  constructor(
+    private readonly service: OracleDiscoveryService,
+    @InjectMapper() private readonly mapper: Mapper,
+  ) {}
+  @ApiTags('Oracle-Discovery')
+  @Get('/oracles')
+  @ApiOperation({ summary: 'Oracles discovery' })
+  @UsePipes(new ValidationPipe())
+  public getOracles(
+    @Body() oracleDiscoveryDto: OracleDiscoveryDto,
```

Comment:
If try to use this with swagger, I get this error "Request with GET/HEAD method cannot have body."